### ISSUE TITLE
Interested in extending PPIx-Refactor to support users in fixing Perl-Critic violations?

### ISSUE THREAD

**Issue by otrosien** [singingfish/PPIx-Refactor#1]
I originally brought up this idea on the Perl-Critic RT, but it was never picked up, as it's not Perl-Critic's core concern. But if you're interested, I have already written a number of refactoring rules at my previous employer, so I know it's doable and worth putting some effort into. 

Link: https://github.com/Perl-Critic/Perl-Critic/issues/375

**Comment by dr-kd** [comment]
PPIx::Refactor came about because of the frustration I had with the interface for Perl::Critic, plus an optional write layer which I never actually used aside as a proof of concept.  So yes I believe for PPIx::Refactor that changes to demonstrate and improve the writing features would be good.

**Comment by phamdt** [comment]
any progress here?

**Comment by otrosien** [comment]
Nope. I personally don't have much interactions with perl anymore, I just know that we once integrated such a refactoring tool at my previous employer. The best part of it was about organizing and fixing missing/obsolete imports. I don't think they will ever open source it, though.
